apps/.eslintrc.js

```diff
@@ -1,51 +1,73 @@
-// This config only defines globals available especially in apps,
-// and enables es6. See the root .eslintrc.js for linting rules.
+// This config defines globals available especially in apps,
+// enables es6, and enables apps-specific plugins and rules.
+// See the root .eslintrc.js for generic eslint linting rules.
 module.exports = {
-  "parser": "babel-eslint",
-  "plugins": ["react"],
-  "parserOptions": {
-    "sourceType": "module",
-    "ecmaFeatures": {
-      "jsx": true,
-      "modules": true,
-      "ecmaVersion": 6,
-      "experimentalObjectRestSpread": true
+  parser: '@babel/eslint-parser',
+  plugins: ['cdo-custom-rules', 'react', 'react-hooks', 'mocha', 'babel'],
+  parserOptions: {
+    babelOptions: {
+      presets: ['@babel/preset-react']
     }
   },
-  "env": {
-    "es6": true
+  extends: ['plugin:react/recommended', 'plugin:prettier/recommended'],
+  env: {
+    es6: true
   },
-  "globals": {
-    "Blockly": true,
-    "Phaser": true,
-    "Studio": true,
-    "Maze": true,
-    "Turtle": true,
-    "Bounce": true,
-    "Eval": true,
-    "Flappy": true,
-    "Applab": true,
-    "Calc": true,
-    "Jigsaw": true,
-    "$": true,
-    "jQuery": true,
-    "IN_UNIT_TEST": true,
-    "IN_STORYBOOK": true,
+  globals: {
+    Blockly: true,
+    Phaser: true,
+    Studio: true,
+    Maze: true,
+    Turtle: true,
+    Bounce: true,
+    Eval: true,
+    Flappy: true,
+    Applab: true,
+    Calc: true,
+    Jigsaw: true,
+    $: true,
+    jQuery: true,
+    IN_UNIT_TEST: true,
+    IN_STORYBOOK: true
   },
   rules: {
-    "react/display-name": "off",
-    "react/jsx-closing-bracket-location": "error", // autofixable
-    "react/jsx-curly-spacing": "error", // autofixable
-    "react/jsx-first-prop-new-line": ["error", "multiline"],
-    "react/jsx-indent-props": ["error", 2], // autofixable
-    "react/jsx-key": "off",
-    "react/jsx-no-target-blank": "error",
-    "react/jsx-wrap-multilines": "error", // autofixable
-    "react/no-find-dom-node": "off",
-    "react/no-render-return-value": "off",
-    "react/no-string-refs": "off",
-    "react/no-unescaped-entities": "off",
-    "react/self-closing-comp": "error",
-    "react/no-danger": "error",
-  }
+    'babel/semi': 'error', // autofixable
+    'cdo-custom-rules/style-blocks-below-class': 'error',
```

Comment:
I see a few rules like this one moved into apps, did that happen as part of the linting or was a conscious choice (I think this one in particular makes sense here b/c its react-specific)?

Reply:
Conscious choice I forgot to include in the PR description! 2 reasons: they are all specific to apps (react, babel, mocha tests), and now eslint does package resolution slightly differently, so us defining the rules in a folder where the packages did not exist (all the eslint packages are installed to `/apps`) was causing some inconsistent behavior.